docs/usage/security/shoot_serviceaccounts.md

```diff
@@ -70,7 +70,6 @@ This ability removes the need for changing the `.spec.kubernetes.kubeAPIServer.s
 Prerequisites:
 - The Garden Cluster should have the Gardener Discovery Server deployed and configured.
   The easiest way to handle this is by using the [gardener-operator](../../concepts/operator.md#gardener-discovery-server).
-- The [`ShootManagedIssuer`](../../deployment/feature_gates.md#list-of-feature-gates) feature gate should be enabled.
```

Comment:
What about the secret labeled with `gardener.cloud/role=shoot-service-account-issuer`, it is also a prerequisite managed by gardener administrator.

Reply:
Yes, but as we only mention the Gardener Operator which is the only supported way to install the Discovery Server as of now I decided to omit this detail since the Gardener Operator handles the creation of the secret when the Discovery Server is configured.